app/grandchallenge/core/templates/challenge.html

```diff
@@ -33,79 +33,78 @@
-                    <li class="nav-item">
-                        <a class="nav-link {% if request.resolver_match.view_name == 'participants:registration-create' %}active{% endif %}"
-                           href="{% url 'participants:registration-create' challenge_short_name=challenge.short_name %}">
-                            Join
-                        </a>
-                    </li>
-                {% endif %}
+                    {% if challenge.display_forum_link %}
+                        <li class="nav-item">
+                            <a class="nav-link"
+                               href="{% url 'forum:forum' slug=challenge.forum.slug pk=challenge.forum.pk %}">
+                                Forum
+                            </a>
+                        </li>
+                    {% endif %}
 
-                {% if challenge.use_evaluation %}
-                    {% if challenge.use_teams %}
-                        {% if "change_challenge" in challenge_perms or user_is_participant %}
-                            <li class="nav-item">
-                                <a class="nav-link {% if request.resolver_match.view_name == 'teams:list' %}active{% endif %}"
-                                   href="{% url 'teams:list' challenge_short_name=challenge.short_name %}">Teams</a>
-                            </li>
+                    {% if challenge.use_evaluation %}
+                        {% if challenge.use_teams %}
+                            {% if "change_challenge" in challenge_perms or user_is_participant %}
+                                <li class="nav-item">
+                                    <a class="nav-link {% if request.resolver_match.view_name == 'teams:list' %}active{% endif %}"
+                                       href="{% url 'teams:list' challenge_short_name=challenge.short_name %}">Teams</a>
+                                </li>
+                            {% endif %}
                         {% endif %}
-                    {% endif %}
-                    {% for phase in challenge.phase_set.all %}
+
                         {% if "change_challenge" in challenge_perms or user_is_participant %}
                             <li class="nav-item">
-                                <a class="nav-link {% if request.resolver_match.view_name == 'evaluation:submission-create' and request.resolver_match.kwargs.slug == phase.slug %}active{% endif %}"
-                                   href="{% url 'evaluation:submission-create' challenge_short_name=challenge.short_name slug=phase.slug %}">Create {{ phase.title }}
-                                    Submission</a>
+                                <a class="nav-link {% if request.resolver_match.view_name == 'evaluation:submission-create' %}active{% endif %}"
+                                   href="{% url 'evaluation:submission-create' challenge_short_name=challenge.short_name slug=challenge.phase_set.first.slug %}">
```

Comment:
Using `challenge.phase_set.first.slug` is good! The phases are always visible, but maybe we need to think about that later.

Reply:
This was a comment, not a request for change. Phases have a start and closing date which are currently unused, I know it's a feature that we want, but not now.

Reply:
gottcha!